render_files.py

```diff
@@ -119,12 +125,28 @@ def prepare_install_config(config_dir, install_config):
             file_obj.write(install_config)
 
 
+def set_pull_secret(config_dir):
+    with open(os.path.join(config_dir, INSTALL_CONFIG), 'r') as json_file:
+        pull_secret = yaml.safe_load(json_file)['pullSecret']
+    with open('/root/.docker/config.json', 'w+') as config_file:
+        config_file.write(pull_secret)
```

Comment:
why not json.dump?

Reply:
because install-config is in yaml format, and not json

Reply:
so yaml.dump

Reply:
but u call file json
so how to are saving yaml there?